### DISCUSSION TITLE
【委托】有人知道绳网手机端怎么整吗？

### DISCUSSION THREAD

**Discussion by zifeiyu1111** [share121/inter-knot#1342] [其他]
外出的时候不方便一直随身带着电脑，想拿手机登录下绳网

**Comment by Inter-Knot-User** [comment]
@share121

**Comment by hjejdjdj** [comment]
在via浏览器搜索share121.github.io/inter-knot/（应该是这个）然后安装绳网助手再登录账号就好了

**Comment by flyhig** [comment]
![Screenshot_2024-08-10-21-46-12-169_mark via](https://github.com/user-attachments/assets/08020ae6-a269-4484-9770-9ff5c5457d33)
下载个加速器就好了。

**Comment by hjejdjdj** [comment]
在via浏览器搜索share121.github.io/inter-knot/（应该是这个）然后安装绳网助手再登录账号就好

**Comment by tuzibuqiahuluobo** [comment]
这个委托。。。算了，不接了
